### ISSUE TITLE
Unable to build on macOS

### ISSUE THREAD

**Issue by pjrader1** [anholt/libepoxy#146]
I'm running macOS (10.13) and I'm having some trouble building libepoxy. I made sure that Meson and Ninja were up to date, as well as Xcode (and the Xcode tools). I'm not overriding any defaults for the Meson configuration. Meson generates fine, but when I run `ninja` I get the following error:
```
...
[9/13] Linking target src/libepoxy.0.dylib.
FAILED: src/libepoxy.0.dylib 
cc  -o src/libepoxy.0.dylib 'src/epoxy@sha/meson-generated_.._gl_generated_dispatch.c.o' 'src/epoxy@sha/dispatch_common.c.o' -shared -install_name /Users/pjrader1/Developer/Git/PGame/deps/libepoxy/libepoxy/build/src/libepoxy.dylib -compatibility_version=1 -current_version=1.0 -ldl '-Wl,-rpath,$ORIGIN/'  
ld: malformed 32-bit x.y.z version number: =1
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```
I tried looking online for a solution but I can't seem to find anything. When I built libepoxy earlier this year (around January) I was able to build it (on macOS 10.12, upgrading OS always has problems so I'm honestly not surprised). I can't seem to figure out what's going wrong and how to fix it.

**Comment by ebassi** [comment]
Which version of libepoxy are you trying to build?

This should have been fixed by issue #108; care to try with a more recent checkout of libepoxy?

**Comment by pjrader1** [comment]
Alright, yes you are correct. I had checked out v1.4.3, which looking at the commit history doesn't include the fix. I had tried checking out the latest commit last night but I got my commands mixed up and didn't actually do it, I accidentally kept trying v1.4.3. It built correctly now. Thanks!
